database/migrations/2021_11_27_124949_create_conversations_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use RonasIT\Support\Traits\MigrationTrait;
+
+class CreateConversationsTable extends Migration
+{
+    use MigrationTrait;
+
+    public function up()
+    {
+        Schema::create('conversations', function (Blueprint $table) {
+            $table->increments('id');
+
+            $table->foreignId('sender_id')->references('id')->on('users')->onDelete('cascade');
```

Comment:
let's move `users` table name to the package config as it implemented in the previous migration